.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,5 +15,9 @@
 - [ ] Tested execution of all binary files (usually in `./result/bin/`)
 - [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
 
+###### Reviewing this pull-request
+
+See the nixpkgs manual for more details on how to [Review contributions](http://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download-by-type/doc/manual#chap-reviewing-contributions).
+
```

Comment:
I'm not sure about putting this in the PR template. It is already pretty noisy. I think linking in in CONTRIBUTING.md is enough. If people don't know about it they can be pointed there by people who do.